ansible/roles/host-ocp4-provisioner/tasks/main.yml

```diff
@@ -11,6 +11,9 @@
 - when: cloud_provider == "ec2"
   import_tasks: "{{cloud_provider}}_prereqs.yml"
 
+- when: cloud_provider == "azure"
```

Comment:
This whole logic feels weird. How about a when cloud_provider=ec2 or azure or osp then {{cloud_provider}}_prereqs.yaml. Or just name it ec2_prereqs.yaml in the import_tasks. I don't know. Just looks odd.

Reply:
I just copy/pasted the line, I will combine them to be a when with an OR.